src/main/java/com/aws/emr/profiler/core/SparkProfiler.java

```diff
@@ -16,9 +16,35 @@
 
 package com.aws.emr.profiler.core;
 
+import com.aws.emr.profiler.cli.Config;
+import com.aws.emr.profiler.core.metricscollector.MetricsCollector;
+import com.aws.emr.profiler.core.metricscollector.SparkMetricsCollector;
+import com.aws.emr.profiler.core.persister.Persister;
+import com.aws.emr.profiler.core.persister.S3Persister;
+
+import java.time.Instant;
+
 public class SparkProfiler implements Profiler {
+
+    private static String FILE_NAME_FORMAT = "emr-profiler-spark-metrics-%tQ";
+
+    private MetricsCollector sparkMetricsCollector;
+    private Persister s3MetricsPersister;
```

Comment:
nit: rename to metricsPersister since it's the interface

Reply:
sure